contracts/placeholder/test/public_api_placeholder.sol

```diff
@@ -41,12 +40,18 @@ contract TestPlaceholderVerifier {
     //  [..., D_omegas_i, ...]
     // 8 + D_omegas_size) q_size
     //  [..., q_i, ...]
-       uint256[] calldata init_params,
-       int256[][] calldata columns_rotations,
-       address gate_argument
-    ) public view{
+    function verify(
+        bytes calldata blob,
+        uint256[] calldata init_params,
+        int256[][] calldata columns_rotations,
+        uint256[] calldata public_input,
+        address gate_argument
+    ) public{
+//        uint256 gas = gasleft();
+        bool b = IVerifier(_verifier).verify(blob,init_params,columns_rotations,public_input,gate_argument); 
+//        console.log("        Gas usage:", gas - gasleft());
```

Comment:
If this is the only profiling we need , i think we can remove the commented code and add it as and when we require.